src/App.js

```diff
@@ -1,7 +1,18 @@
+import Activity_card from './Components/Activity_card/Activity_card';
```

Comment:
`Activity_card` meglio chiamarla `ActivityCard`

anche il file meglio in PascalCase se si tratta di componenti React 

Nota i suggerimenti che vengono dati in console